### PR TITLE
Prepare the artefact

### PR DESCRIPTION
**Pull request by ptersilie**
This PR gets the repo into shape for the artefact. One thing I'm currently unsure about is if I should add the logs from the paper to the repo as well (they are needed to reproduce the exact same results as in the paper) or if I should only add that to the artefact VM later on.

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -0,0 +1,35 @@
+mkdir logs
+
+echo "Running expression benchmarks"
```
Comment (ltratt) [inline build.sh]: Conventionally we tend to prefix these informational lines with `===>`  so that they stand out from other output.

**Comment by ltratt** [comment]
> One thing I'm currently unsure about is if I should add the logs from the paper to the repo as well

We shouldn't add those here IMHO, but we should add them (separately) on archive when we eventually upload things (see e.g. the error recovery experiment) as well as in the VM.

**Comment by ptersilie** [comment]
Okay, that matches what I had in mind. :+1:

File: build.sh (hunk 1 above)
Comment (ptersilie) [reply build.sh]: Fixed.

**Review by ltratt — APPROVED** [review]

**Comment by ltratt** [comment]
Please squash.

**Comment by ptersilie** [comment]
Squashed!

**Pull request merged**